lib/icalendar/alarm.ex

```diff
@@ -0,0 +1,21 @@
+defmodule ICalendar.Alarm do
+  @moduledoc """
+  Events have alarms.
+  """
+
+  defstruct minutes_before: 60
```

Comment:
No, they can be specified in many different ways. The `minutes_before` attribute is very narrow and abstract in contrast to the possibilities of the iCal standard, which allows alarm triggers to be set to an absolute time or be relative to the event (from seconds to weeks).

I guess I've chosen this field because it fits my current requirements 😄 

* [Alarm definition](https://tools.ietf.org/html/rfc5545#section-3.6.6)
* [Duration definition](https://tools.ietf.org/html/rfc5545#section-3.3.6) as used for relative triggers.